integration_tests/integration/barcode/createBarcodeAddressImage.spec.ts

```diff
@@ -21,4 +22,14 @@ context('Create Barcode Image', () => {
     const page = Page.verifyOnPage(ChooseBarcodeOptionPage)
     page.continueToImageErrors().hasErrorContaining('error')
   })
+
+  it('should show an error if creating the first barcode succeeds but subsequent barcodes fail', () => {
+    cy.task('stubCreateBarcodeSuccessThenCreateBarcodeFailure')
+    cy.go(-1) // back to Review Recipients in order to add a 2nd recipient
```

Comment:
Is the add recipient link also tested elsewhere?

Reply:
Yes, as part of `legalSenderJourneyE2e.spec.ts` 👍